src/main/java/org/betonquest/betonquest/VariableNumber.java

```diff
@@ -135,6 +135,16 @@ private double resolveVariable(final Profile profile) {
         }
     }
 
+    /**
+     * To check if a value will be guarantied not zero or less
+     *
+     * @return true if a variable is set or the constant value is greater zero,
+     * false if no variable is set and the constant value is zero or less
+     */
+    public boolean explicitLessThanOne() {
```

Comment:
could add an "is"to the name of the method

Reply:
Yes it should be `isExplicitLessThanOne`